cityofaustin/vision-zero
# The is_manual_geocode column is not correct when CRIS values are null

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/19723

## Testing

**URL to test:** 
Local

**Steps to test:**
1. Apply the migrations
2. See that this crash now has an is_manual_geocode value of true
```sql
SELECT
    crash.latitude lat_crash,
    crash.longitude lon_crash,
    cris.latitude lat_cris,
    cris.longitude lon_cris,
    crash.is_manual_geocode
FROM
    crashes_list_view crash
    LEFT JOIN crashes_cris cris ON crash.id = cris.id
WHERE
    crash.cris_crash_id = 16318132;
```
3. Test editing the coordinates of a crash and see that the is manual geocode column updates accordingly
4. Test the down migration

---

#### Ship list
